FirstAlert ZCOMBO Smoke Alarm

parasthakkar-newell opened the pull request:
Integration of 500 series

parasthakkar-newell committed:
FirstAlert ZCOMBO Smoke Alarm

Integration of 500 series

tpmanley commented on devicetypes/smartthings/zwave-smoke-alarm.src/zwave-smoke-alarm.groovy:
```diff
@@ -21,7 +21,7 @@ metadata {
 		attribute "alarmState", "string"
 
-		fingerprint mfr:"0138", prod:"0001", model:"0002", deviceJoinName: "First Alert Smoke Detector and Carbon Monoxide Alarm (ZCOMBO)"
+		fingerprint mfr:"0138", prod:"0001", model:"0003", deviceJoinName: "First Alert Smoke Detector and Carbon Monoxide Alarm (ZCOMBO)"
```
Thanks for the PR. Can you leave the existing fingerprint and just add the new one?

parasthakkar-newell commented:
I need to add new one.
Leave the existing one as it is.

From: Tom Manley <notifications@github.com>
Sent: Thursday, October 24, 2019 10:01 AM
To: SmartThingsCommunity/SmartThingsPublic <SmartThingsPublic@noreply.github.com>
Cc: Thakkar, Paras <Paras.Thakkar@Newellco.com>; Author <author@noreply.github.com>
Subject: Re: [SmartThingsCommunity/SmartThingsPublic] FirstAlert ZCOMBO Smoke Alarm (#11464)


@tpmanley commented on this pull request.

________________________________

In devicetypes/smartthings/zwave-smoke-alarm.src/zwave-smoke-alarm.groovy<https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/11464#discussion_r338625612>:

> @@ -21,7 +21,7 @@ metadata {



               attribute "alarmState", "string"



-              fingerprint mfr:"0138", prod:"0001", model:"0002", deviceJoinName: "First Alert Smoke Detector and Carbon Monoxide Alarm (ZCOMBO)"

+              fingerprint mfr:"0138", prod:"0001", model:"0003", deviceJoinName: "First Alert Smoke Detector and Carbon Monoxide Alarm (ZCOMBO)"

Thanks for the PR. Can you leave the existing fingerprint and just add the new one?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/11464?email_source=notifications&email_token=AKMJLXPA7X2IDK56KHUZM73QQG2BDA5CNFSM4JEU6JQ2YY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCJDQQTA#pullrequestreview-306645068>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AKMJLXJDMFJPF5NO3JDCILDQQG2BDANCNFSM4JEU6JQQ>.

tpmanley commented:
@parasthakkar-newell do you have the new version of this device? I don't see it on the Z-Wave certified product list

parasthakkar-newell commented:
Tom,

Certification is in progress at Pepper-one lab.

Thanks,
Paras


From: Tom Manley <notifications@github.com>
Sent: Thursday, October 24, 2019 11:14 AM
To: SmartThingsCommunity/SmartThingsPublic <SmartThingsPublic@noreply.github.com>
Cc: Thakkar, Paras <Paras.Thakkar@Newellco.com>; Mention <mention@noreply.github.com>
Subject: Re: [SmartThingsCommunity/SmartThingsPublic] FirstAlert ZCOMBO Smoke Alarm (#11464)


@parasthakkar-newell<https://github.com/parasthakkar-newell> do you have this device? I don't see it on the Z-Wave certified product list

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/11464?email_source=notifications&email_token=AKMJLXOR7CEIYD5BPVDE6J3QQHCVJA5CNFSM4JEU6JQ2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOECFS3IA#issuecomment-545992096>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AKMJLXNGUJH7KSEY4LRGSA3QQHCVJANCNFSM4JEU6JQQ>.

parasthakkar-newell committed:
Updated to not remove older product

Since new and old both products are in market at the same time, I have added old model back in this file. Please review.

parasthakkar-newell commented:
Got these changes merged in another pull request https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/12335, so closing this one
